src/LodeRunner/src/Services/LodeRunnerService.cs

```diff
@@ -365,5 +380,31 @@ private void RegisterCancellationTokensForServices()
                 this.GetClientStatusService().TerminateService(this.clientStatus);
             });
         }
+
+        /// <summary>
+        /// Called when [status timer event].
+        /// </summary>
+        /// <param name="sender">The sender.</param>
+        /// <param name="args">The <see cref="ElapsedEventArgs"/> instance containing the event data.</param>
+        private void OnStatusTimerEvent(object sender, ElapsedEventArgs args)
+        {
+            this.lastStatusArgs.LastUpdated = DateTime.UtcNow;
+            StatusUpdate(this.lastStatusSender, this.lastStatusArgs);
+        }
```

Comment:
Should "args" and "sender" be passed here to StatusUpdate instead of lastStatusArgs and lastStatusSender? I may be missing something as I am not very familiar with the code base, but it looks like in StatusUpdate below, lastStatusArgs and lastStatusSender are reassigned to the same values.

Reply:
The `args` input is not being passed to StatusUpdate() because it is not an instance of the right class (ClientStatusEventArgs).  ClientStatusEventArgs is used here, because the LastUpdated property needs to be updated. The `sender` argument is also not used and I've put a triage item on the board so we can discuss why this was implemented this way. It's currently set to its initial value (null), and never updated.